pkgs/applications/networking/netclient/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, lib
+, libglvnd
+, pkg-config
+, xorg
+}:
+
+buildGoModule rec {
+  pname = "netclient";
+  version = "0.20.3";
+
+  src = fetchFromGitHub {
+    owner = "gravitl";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-GeXmAwmGbb73Z0OhW7cNu6UkccpaHqVQOItEK58Jh1o=";
+  };
+
+  vendorHash = "sha256-qchVMLmdvqpVpyjDy05W8yHTBcYCgpyuX5Ibg+yceXY=";
+
+  buildInputs = [
+    libglvnd
+    xorg.libX11
+    xorg.libXcursor
+    xorg.libXi
+    xorg.libXinerama
+    xorg.libXrandr
+  ];
+
+  postInstall = ''
+    install -Dm444 build/netclient.service -t $out/lib/systemd/netclient.service
```

Comment:
The -t flag copies all SOURCE arguments into DIRECTORY

Either drop that flag
`install [OPTION]... SOURCE... DIRECTORY`

Or rearrange the args
`install [OPTION]... -t DIRECTORY SOURCE...`

Reply:
Can be ignored, or remove postInstall

Reply:
I've remove it, since it's not needed